sqlitecluster/SQLiteNode.cpp

```diff
@@ -2580,6 +2541,77 @@ void SQLiteNode::_recvSynchronize(Peer* peer, const SData& message) {
         throw "commits remaining at end";
 }
 
+bool SQLiteNode::peerClosenessSort(const Peer* lhs, const Peer* rhs)
+{
+    // Null pointers are never fast/close.
+    if (!lhs) {
+        return false;
+    }
+    if (!rhs) {
+        return true;
+    }
+
+    // If two peers have the same latency, then the one with the higher CommitCount is closest. This case is most
+    // likely in the event that we have no latency data for either peer.
+    if (lhs->latency == rhs->latency) {
+        return SToInt((*lhs)["CommitCount"]) > SToInt((*rhs)["CommitCount"]);
```

Comment:
You could also do `lhs->calc("CommitCount")` here, because a `Peer` extends `SData`.